pilot/test/integration/testdata/rule-default-route-mirrored.yaml.tmpl

```diff
@@ -1,14 +1,15 @@
 apiVersion: config.istio.io/v1alpha2
-kind: RouteRule
+kind: V1alpha2RouteRule
 metadata:
-  name: default-route
+  name: route-rule-c
 spec:
-  destination:
-    name: c
-  precedence: 1
-  route:
-    - labels:
-         version: v1
-      weight: 100
-  mirror:
-    name: b
+  hosts:
+    - c
+  http:
+    - route:
+      - destination:
+          name: c
+          subset: v1
+        weight: 100
+      mirror:
+        name: b
```

Comment:
This relates to @christian-posta 's bug (https://github.com/istio/istio/issues/3270).. We shouldn't require mirror

Reply:
Agreed. This is due to a bug in the implementation of shadow cluster that was copied over from v1alpha1. Should be an easy fix.